### ISSUE TITLE
CSS is Not working on Web.

### ISSUE THREAD

**Issue by kiritnim** [kristerkari/react-native-css-modules#31]
Hi,

I have integration as follow the steps you provided but the after creating CSS file and apply on js file. it's working fine in the iOS device but not working on web.

**My Code :** 

```
import React, { Component } from 'react'
import { ScrollView, StyleSheet, TouchableOpacity, Text, View } from 'react-native'
import styles from "./styles.css";


export default class Dashboard extends React.Component {
    render() {
        return (
            <View className={styles.container}>
                <Text className={styles.blue}>Blue Text</Text>
            </View>
        )
    }
}
```
OnWeb : 
![Screenshot 2019-10-03 at 10 08 57 AM](https://user-images.githubusercontent.com/17870723/66099844-1d566f80-e5c6-11e9-8dbf-c647fc38f02b.png)

OnIOS:
![Screenshot 2019-10-03 at 10 09 44 AM](https://user-images.githubusercontent.com/17870723/66099856-27786e00-e5c6-11e9-8912-5ad2564f735d.png)

Can you help me.

Thanks.

**Comment by kristerkari** [comment]
hey @kiritnim! 

I think that the problem is that the Web part is not properly set up. Do you have your project somewhere where I could have a look at it? For example a new Github repository?

Here are the docs for the Web:
https://github.com/kristerkari/react-native-css-modules/blob/master/docs/web-compatibility.md

...and here are the needed files/config from the example app:
- Webpack config: https://github.com/kristerkari/react-native-css-modules-example/blob/master/webpack.config.js
- Web index.js: https://github.com/kristerkari/react-native-css-modules-example/blob/master/index.web.js
- Web index.html: https://github.com/kristerkari/react-native-css-modules-example/blob/master/index.html

**Comment by kiritnim** [comment]
> hey @kiritnim!
> 
> I think that the problem is that the Web part is not properly set up. Do you have your project somewhere where I could have a look at it? For example a new Github repository?
> 
> Here are the docs for Web:
> https://github.com/kristerkari/react-native-css-modules/blob/master/docs/web-compatibility.md
> 
> ...and here are the needed files/config from the example app:
> 
> * Webpack config: https://github.com/kristerkari/react-native-css-modules-example/blob/master/webpack.config.js
> * Web index.js: https://github.com/kristerkari/react-native-css-modules-example/blob/master/index.web.js
> * Web index.html: https://github.com/kristerkari/react-native-css-modules-example/blob/master/index.html

I am use Expo, and the app is start with App.js, So can you help me with expo app.

**Comment by kiritnim** [comment]
I am use Expo, and the app is start with App.js, So can you help me with expo app.

**Comment by kristerkari** [comment]
I see, I have not tested the Web compatibility with Expo. I can have a look at it and update the docs to have the instructions for Expo too.

**Comment by kiritnim** [comment]
Okay. Thanks I wait for it.

**Comment by yyaswanth** [comment]
Hi @kristerkari 

Even i am facing the same issue. I am using the expo. For web css styles are not loading but for mobile ( ios & android ) styles are loading properly. Could you please add the solution.

**Comment by RobinS-S** [comment]
Has anyone found a solution yet?

**Comment by PPillau** [comment]
Would be interested too. Any way of getting this to work in Expo Web?

**Comment by kristerkari** [comment]
@yyaswanth @RobinS-S @PPillau I'm not using Expo myself, but I know that adding the support is a bit trickier at the moment since [react-native-web](https://github.com/necolas/react-native-web) dropped the support for using the `className` property.

What would probably be required to get it working with Expo Web:
1. Patch `react-native-web` to bring back support for the `className` property.
2. Add Webpack configuration to Expo that would use the Webpack (Web) css modules.

**Comment by sikaco** [comment]
So, is there anyone know how to solve it?

I use expo too, just follow the path from react-native official. 😂

**Comment by JoeBoulton** [comment]
Any update on this?

**Comment by kristerkari** [comment]
> Any update on this?

Yes, please check https://github.com/kristerkari/react-native-css-modules/issues/37#issuecomment-1269678996
